common/setups/serialization.py

```diff
@@ -95,7 +102,14 @@ def __init__(
 
         self.object_name = self.code_object.split(".")[-1]
         self.module = ".".join(self.code_object.split(".")[:-1])
-        self.package = ".".join(self.code_object.split(".")[:-2])
```

Comment:
You are completely removing `self.package` here. In my setup I was using a [Collection](https://github.com/rwth-i6/i6_experiments/blob/8f65f607a9a159cfa0b98df2f13be889760d30a3/common/setups/returnn_pytorch/serialization.py#L69). The docstring said about the `package` argument:

https://github.com/rwth-i6/i6_experiments/blob/8f65f607a9a159cfa0b98df2f13be889760d30a3/common/setups/returnn_pytorch/serialization.py#L96

Which turns out was a lie. So instead I was giving the `some_import.package` of all my `Import`s. 

What is your advice to do instead? would giving `some_import.module` work? The code seems to expect to copy a folder and not a file. But anyway if I just need one module I would prefer to only copy one file..

Reply:
The code worked liked that in the beginning, but I found it to be a suboptimal solution. Thus I started copying folders instead, so that I could share some common code. Maybe we do not want that? 

> So instead I was giving the some_import.package of all my Imports.

This one I do not understand...

Reply:
Would just keeping `self.package` be okay for you?

Reply:
>Would just keeping self.package be okay for you?

This would work for me. 

Or I just copy the line 
```
package = ".".join(my_import.code_object.split(".")[:-2])
```
in my recipe collection if you think `self.package` should not me here.